### DISCUSSION TITLE
Tile strangeness

### DISCUSSION THREAD

**Discussion by jturnley** [ssitu/ComfyUI_UltimateSDUpscale#116] [Q&A]
I set up my workflows to use a  2x2 uniform tile layout with each tile slightly larger than 1/4 the size of the final image to improve output quality  (i.e. so the  model has a better chance of seeing enough of a face to not turn the output into  a  Lovecraftian horror). When it gets to the 4th tile, rather than being uniform in size, it picks a small section in the bottom right corner and renders that instead. Usually this doesn't cause a problem, because seam fix goes back and fixes it, but occasionally decides something down in that corner is a face or the like because the area is too small for the model to get enough context, and messes up the whole workflow.

Does anyone know, is this a bug, or is there a way around this issue?

**Comment by alex-mitov** [comment]
I have the same problem!
